File: 18-Problem-RepeatSixteenAgain/18_Problem_RepeatSixteenAgain.py
# Repeat problem 16 again to work rote memory of basic syntax.
import random

NINT = random.randint(1, 10)
FLOTE = 32.32
CHR = 'c'
STR = "String"
BOO = True

def increment_NINT(num):
    if num > 5:
        adder = random.randint(1, 10)
        loops = random.randint(1, 10)

        while loops > 0:
            num+= adder
            adder+=1
            loops-=1

    return num

print(f"NINT: {NINT} FLOTE: {FLOTE} CHR: {CHR} STR: {STR} BOO: {BOO}")
print(f"increment_NINT(NINT): {increment_NINT(NINT)}")

File: 18-Problem-RepeatSixteenAgain/18-Problem-RepeatSixteenAgain.js
// Repeat problem 16 again to work rote memory of basic syntax.

const NINT = Math.floor(Math.random() * (10 - 1) + 1);
const FLOTE = 32.32;
const CHR = 'c';
const STR = 'String';
const BOO = true;

const incrementNint = (num) => {
  if(num > 5) {
    let adder = Math.floor(Math.random() * (10 - 1) + 1);
    let loops = Math.floor(Math.random() * (10 - 1) + 1);

    while(loops > 0) {
      num+= adder;
      adder++;
      loops--;
    }
  }
  return num;
}

console.log(`NINT: ${NINT} FLOTE: ${FLOTE} CHR: ${CHR} STR: ${STR} BOO: ${BOO}`);
console.log(`incrementNint(NINT): ${incrementNint(NINT)}`);


